features/init.feature

```diff
@@ -6,3 +6,11 @@ Feature: Initialize a new project using Tuist
     When I initialize a macos application named Test
     Then tuist generates the project
     Then I should be able to build the scheme Test
+
+  Scenario: The project is a compilable iOS application
```

Comment:
Same comment here regarding merging this test into an existing one adding the extra necessary steps.